setup.py

```diff
@@ -64,6 +64,6 @@ def get_requirements() -> Sequence[str]:
         ],
         "libsentrykube.macros": macros,
     },
-    scripts=["bin/sentry-kube-pop"],
+    scripts=["bin/sentry-kube-pop", "sentry_kube/bin/important-diffs-only"],
```

Comment:
This adds the binary as part of the python package

Reply:
Should we consolidate to either `sentry_kube/bin` or `bin/`?

Reply:
sentry-kube-pop wraps sentry-kube while important-diffs-only is used from within sentry-kube (it also looks like sentry-kube-pop may not work with the new regional clusters).

Reply:
Consolidated to sentry_kube/bin

Reply:
> (it also looks like sentry-kube-pop may not work with the new regional clusters).

Is it because of moving things out of the ops repo? Or it it because of something else

Reply:
Yeah, I use my own wrappers..

```
function skpop() {
  echo "sentry-kube -C ${1} -c ${2} -q ${@:3}"
  sentry-kube -C ${1} -c ${2} -q ${@:3}
}

function skpopus() {
  regions=(us europe asia australia)
  for region in $regions ; do echo "### REGION: ${region} ###" ; echo "sentry-kube -C saas -c pop-${region}-regional -q ${@:1}" ; sentry-kube -C saas -c pop-${region}-regional -q ${@:1} ; done
}

function skpopde() {
  regions=(us europe)
  for region in $regions ; do echo "### REGION: ${region} ###" ; echo "sentry-kube -C de -c pop-${region}-regional -q ${@:1}" ; sentry-kube -C de -c pop-${region}-regional -q ${@:1} ; done
}
```